tool/jt.rb

```diff
@@ -43,6 +43,22 @@
 METRICS_REPS = Integer(ENV['TRUFFLERUBY_METRICS_REPS'] || 10)
 DEFAULT_PROFILE_OPTIONS = %w[--cpusampler --cpusampler.Output=flamegraph]
 
+MRI_TEST_RETAG_RETAIN_PATTERNS = [
+  /hangs/i,
+  /slow/i,
+  /java\.lang\./,
+  /retain-on-retag/,
+  /OOM/,
+  /RubyVM/,
+  /ShouldNotReachHere/,
```

Comment:
I'm not sure if we should retain these, I guess the main thing is it makes it much faster to retag?
But at the price of not checking if those got solves somehow.
There seems to be a single match, so I'd suggest just removing it.

Reply:
I think the problem was it would cause the JVM to exit, leaving the tags file in a bad state. With the new retry logic this shouldn't be a problem. I'll remove it.